dao/src/main/java/greencity/repository/impl/EventSearchRepoImpl.java

```diff
@@ -137,7 +137,7 @@ private void addCitiesPredicate(List<String> cities, Root<Event> eventRoot, List
         if (cities != null && !cities.isEmpty()) {
             Join<EventDateLocation, Address> addressJoin = eventRoot
                 .join(Event_.dates, JoinType.LEFT).join(EventDateLocation_.address);
-            predicates.add(addressJoin.get(Address_.CITY_EN).in(cities));
+            predicates.add(criteriaBuilder.upper(addressJoin.get(Address_.CITY_EN)).in(cities));
```

Comment:
it is also a good idea to make cities be in upper case too

Reply:
Ok.